docs/API.md

```diff
@@ -209,42 +207,34 @@ class MyPlayerBar extends TrackPlayer.ProgressComponent {
             </View>
         );
     }
-    
+
 }
 ```
 
-Note: `formatTime` and `ProgressBar` are just examples, replace them with your own implementation.
-
-Be careful, as the component will be re-rendered every progress update!
+## Track Player Options
 
-You can find more information about the ProgressComponent in the [documentation](https://react-native-kit.github.io/react-native-track-player/documentation/#progresscomponent).
+Track Player can be configured using a number of options. Some of these options pertain to the media controls available in the lockscreen / notification and how they behave, others describe the availability of capabilities needed for platform specific functionalities like Android Auto.
 
-## Options
-You can define the rating type, capabilities, icons, etc.
-Every parameter is optional, you don't have to set all of them.
-You can also call this function multiple times, you don't need to specify all the options, just the ones you want to change.
+You can change options multiple times. You do not need to specify all the options, just the ones you want to change.
 
-It's recommended to set the options right before using any other function (other than `setupPlayer`)
+It is recommended to set the options right before using any other function (other than `setupPlayer`)
```

Comment:
@guichaguri, can you provide any background on this recommendation? It seems to suggest doing it at other times could cause problems?

Reply:
I think it was to prevent multiple metadata updates, as the module would show the notification as soon as it starts playing, but it would have to update the notification right after because of the `setOptions` call.

That can be removed or moved into the formal documentation

Reply:
I opted to remove this.